certbot-apache/certbot_apache/tests/configurator_test.py

```diff
@@ -115,6 +115,20 @@ def test_add_parser_arguments(self):  # pylint: disable=no-self-use
         # Weak test..
         ApacheConfigurator.add_parser_arguments(mock.MagicMock())
 
+    def test_docs_parser_arguments(self):
+        os.environ["CERTBOT_DOCS"] = "1"
+        from certbot_apache.configurator import ApacheConfigurator
+        mock_add = mock.MagicMock()
+        ApacheConfigurator.add_parser_arguments(mock_add)
+        expected = {'default': '/etc/apache2',
+                    'help': 'Apache server root directory'}
+        found = False
+        for call in mock_add.call_args_list:
+            if call[0] == ('server-root',):
+                if call[1] == expected:
+                    found = True
+        self.assertTrue(found)
```

Comment:
I think it's worth adding some more tests here to prevent regressions. Plenty of systems default server root is `/etc/apache2`.

Maybe we can loop thorough all values given to `add` and check all relevant values against `ApacheConfigurator.OS_DEFAULTS`?

What do you think?

Reply:
Yeah, that sounds good